src/types.ts

```diff
@@ -238,6 +238,17 @@ export interface UIDisputeTokens {
   [stakeToken: string]: UIDisputeTokenInfo;
 }
 
+export interface StakeDetails extends Payout {
+  totalStaked: number;
+  size: number;
+  amountStaked: number;
+}
+
+export interface UIStakeInfo {
+  marketID: Address;
+  stakes: Array<StakeDetails>;
+}
```

Comment:
Is there a mapping to Market Outcome and payout? In UI there is just outcomes and TODOs for mapping market payoutNumerators to outcomes.

Reply:
under stakes, there will be payoutNumerators, so i think you'd have to see which one was >0 and which one was == 0?

Reply:
discussion in #dev discord